source/containers.tex

```diff
@@ -2518,6 +2518,11 @@
 \tcode{map}
 and
 \tcode{multimap}.
+The library also provides container adaptors
+that make it easy to construct abstract data types,
+such as \tcode{flat_map}s or \tcode{flat_multimap}s,
+out of the basic sequence container kinds
+(or out of other program-defined sequence containers).
```

Comment:
```suggestion
(or out of other kinds of sequence containers that the user defines).
```
maybe keep it as the same as [sequence.reqmts]?

Reply:
@jwakely Thoughts? Maybe we should change [sequence.reqmts] to also say "program-defined"?

Reply:
I'm making that change; the original wording is from Day 1 and long predates the introduction of the term "program-defined". Thanks!